schema/revert/tables/gas_002_ar5_gwp.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ggircs-portal:tables/gas_002_ar5_gwp from pg
+
+BEGIN;
+
+-- XXX Add DDLs here.
```

Comment:
Makes sense to me, just want to make sure we're intentionally leaving this as a one way mutation

Reply:
Nop, not intentional. Thanks!